src/formatDate.js

```diff
@@ -8,7 +8,56 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separatorFrom = fromFormat.pop();
+  const separatorTo = toFormat.pop();
+
+  const dateParts = date.split(separatorFrom);
```

Comment:
It's a good practice to handle the scenario where the split operation does not return the expected number of date parts. Since the input is assumed to be correctly formatted, this isn't necessary here, but keep this in mind for more robust error handling in future projects.